### ISSUE TITLE
Support JSON log format in faro.receiver

### ISSUE THREAD

**Issue by sigurdfalk** [grafana/alloy#1973]
### Request

Currently, the only available logging format supported by the Faro Receiver is logfmt. JSON is a widely adopted standard for logging and is also the only structured format supported in addition to logfmt in LogQL queries. 

Ref: https://grafana.slack.com/archives/C01050C3D8F/p1728549514784879

### Use case

Our concrete use-case is the we use the [faro.receiver](https://grafana.com/docs/alloy/latest/reference/components/faro/faro.receiver/) which uses logfmt , but our developers complains that this makes combined searches hard for them because everything else we have in our company uses json as log format.

**Comment by ravishankar15** [comment]
Hi Team, I would like to work on this issue, I can see logs, traces and metrics exported from receiver. I think this format is applicable only for logs ? I am hoping the format option is only applicable for logs ? Correct me if my understanding is incorrect

**Comment by sigurdfalk** [comment]
@ravishankar15 Correct, this is only applicable to logs being able to be exported as JSON 👍🏻
